remc2/portability/port_sdl_joystick.cpp

```diff
@@ -543,6 +551,9 @@ void gamepad_event_mgr(gamepad_event_t *gpe)
 		if (gpe->btn_pressed & (1 << gpc.button_minimap)) {
 			setPress(false, GP_KEY_EMU_MINIMAP);
 		}
+		if (gpe->btn_pressed & (1 << gpc.button_options_menu)) {
```

Comment:
this should be 
```
setPress(false, GP_KEY_EMU_OPTIONS);
```

^ false since you are releasing the key

Reply:
I cut and pasted, my bad. Amazingly it still worked